docs/user/openstack/README.md

```diff
@@ -44,7 +83,9 @@ enough to store the ignition config files, so they are served by swift instead.
 * You may need to increase the security group related quotas from their default
   values. For example (as an OpenStack admin) `openstack quota set --secgroups 100 --secgroup-rules 1000 <project>`
```

Comment:
@iamemilio one more thing: this `openstack quota set --secgroups 100 --secgroup-rules 1000 <project>` command here is now inconsistent with the quotas you've added above.

Could you please either delete this line or fix the values and incorporate it into your changes?